BryantUniversity/BryantUniversity/Controllers/FacultyController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BryantUniversity.Data;
+using BryantUniversity.Models;
+using BryantUniversity.Models.Repo;
+using BryantUniversity.ViewModels;
+using System.Web.Mvc;
+
+namespace BryantUniversity.Controllers
+{
+    public class FacultyController : Controller
+    {
+        private Context context;
+
+        public FacultyController()
+        {
+            context = new Context();
+        }
+
+        // GET: Faculty
+        public ActionResult Index()
+        {
+            return View();
+        }
+
+        public ActionResult Create()
+        {
+            var userViewModel = new UserViewModel();
+            return View(userViewModel);
+        }
+
+        [HttpPost]
+        public ActionResult Create(UserViewModel userViewModel)
```

Comment:
You should add a `[ValidateAntiForgeryToken]` attribute here because you have an anti-forgery token in your Create view form.